test/parse.jl

```diff
@@ -298,6 +298,8 @@ end
     @test eltype([tryparse(Int, s, base=16) for s in String[]]) == Union{Nothing, Int}
     @test eltype([tryparse(Float64, s) for s in String[]]) == Union{Nothing, Float64}
     @test eltype([tryparse(Complex{Int}, s) for s in String[]]) == Union{Nothing, Complex{Int}}
+    # JET.@test_opt parse(UInt8, "12")
```

Comment:
Leftovers?

Reply:
Kind of. But I actually want to integrate JET into Julia Base soonish and enable it within a test suite. I'd like to turn on these leftovers then if people aren't unhappy with them.